components/expenses/Order.js

```diff
@@ -222,7 +226,14 @@ class Order extends React.Component {
           </div>
           <OrderDetails order={order} mode={mode} />
           {order.status === 'PENDING' && canMarkOrderAsPaid && (
-            <MarkOrderAsPaidBtn order={order} collective={order.collective} />
+            <Flex>
+              <MarkOrderAsPaidBtn order={order} collective={order.collective} />
+              <CancelPendingOrderBtn
```

Comment:
Feels like we don't really need a dedicated component for this, there's no special logic in `CancelPendingOrderBtn` and it's not especially verbose. Could be re-written inline with the same amount of lines:

```jsx
    <StyledButton bg="red.500" color="#fff" onClick={() => onClickCancel(order.id)}>
      <FormattedMessage id="order.pending.cancel" defaultMessage="Cancel" />
    </StyledButton>
```

But that's a personal preference.